interface/patient_file/encounter/encounter_top.php

```diff
@@ -18,8 +18,10 @@
     if ($set_pid && $set_pid != $_SESSION["pid"]) {
         setpid($set_pid);
     }
-
-    setencounter($_GET["set_encounter"]);
+    $set_encounter = filter_var($_GET["set_encounter"],FILTER_VALIDATE_INT);
+    if ($set_encounter) {
+        setencounter($set_encounter);
+    }
```

Comment:
I think it would make more sense to force this into an `int` in the library/encounter.inc script itself. This is what is done with the pid processing. Then this would be centralized and easy to adjust if needed in the future.

Reply:
Done.